R/parse_api.R

```diff
@@ -0,0 +1,28 @@
+parse_api = function(stem, country) {
+	sane = all(c('RJSONIO', 'RCurl') %in% row.names(installed.packages()))
+	if (!sane) {
+		stop('The RJSONIO and RCurl packages must be installed to parse
+			  countries using online APIs (Google or DSTK).')
+	}
+	url = utils::URLencode(paste0(stem, country))
+	dat = RJSONIO::fromJSON(url, simplify = TRUE)
+	if (dat$status == 'OK') {
+		out = dat$results[[1]]$address_components[[1]]$long_name
+	} else {
+		out = NA
+		warning(paste('Failed to parse country name using the Data Science Tool Kit: ', country, '\n'))
```

Comment:
this warning needs to be less specific because it could apply to Google or DSTK responses